packages/examples/cvat/exchange-oracle/src/cvat/api_calls.py

```diff
@@ -509,13 +509,13 @@ def delete_cloudstorage(cvat_id: int) -> None:
             raise
 
 
-def fetch_projects(assignee: str = "") -> List[models.ProjectRead]:
+def fetch_projects(assignee: str = "") -> list[models.ProjectRead]:
     logger = logging.getLogger("app")
     with get_api_client() as api_client:
         try:
             return get_paginated_collection(
                 api_client.projects_api.list_endpoint,
-                **(dict(assignee=assignee) if assignee else {}),
+                **({"assignee": assignee} if assignee else {}),
```

Comment:
While correct, the previous variant is useful in the sense that it makes changing from a dict to a class as simple as `dict` renaming the class name. With {} you actually need to replace all the quotes and : to make such a transition.

Reply:
I see your point! Fortunately, `ruff` can take care of this case:

as per `PIE804`, this code
```py
MyClass(**{"aws_access_key_id": access_key})
```

automatically gets converted to
```py
MyClass(aws_access_key_id=access_key)
```

So developer won't have to deal with replacing quotes or braces by hand.

Reply:
Nice, great feature.